code/datums/skills/uscm.dm

```diff
@@ -6,7 +6,12 @@ United States Colonial Marines
 
 /datum/skills/pfc
 	name = "Private"
+	skills = list(
+		SKILL_FIREMAN = SKILL_FIREMAN_TRAINED,
+	)
 	//same as default
+	// I added fireman skills here. I suspect that since shooting skills aren't here, they are in the 'default' area, but I don't know where that is
+	// so I've just pasted the fireman skill across every role in the USCM category instead.
```

Comment:
This doesn't need to change. As far as why they don't specify a starting level for say `/datum/skill/firearms` that's because that datum has `skill_level` start at `SKILL_FIREARMS_TRAINED`.

Reply:
Ah, thanks! I was confused by that. Reverted!